configure.ac

```diff
@@ -963,6 +963,78 @@ then
 fi
 
 
+# KYBER
+# Used:
+#  - SHA3, Shake128 and Shake256, or
+#  - SHA256, SHA512, AES-CTR
+AC_ARG_ENABLE([kyber],
+    [AS_HELP_STRING([--enable-kyber],[Enable KYBER (default: disabled)])],
+    [ ENABLED_KYBER=$enableval ],
+    [ ENABLED_KYBER=no ]
+    )
+
+ENABLED_WC_KYBER=no
+for v in `echo $ENABLED_KYBER | tr "," " "`
+do
+  case $v in
+  yes | all)
+    ENABLED_KYBER512=yes
+    ENABLED_KYBER768=yes
+    ENABLED_KYBER1024=yes
+    ;;
+  no)
+    ;;
+  wolfssl)
```

Comment:
I think this should default to the wolfssl implementation. When using it the only config needed for a basic build would be --enable-kyber and if overriding the basic build then need the extra arguments.

Reply:
We can't default to the wolfssl implementation.
Anthony will implement using liboqs or other APIs.

Reply:
Ah, Ok.  I think I understand.  I'm currently working on this.